sdk/src/test/java/com/google/cloud/dataflow/sdk/io/BigQueryIOTest.java

```diff
@@ -1214,6 +1214,84 @@ public void testBigQueryQuerySourceInitSplit() throws Exception {
   }
 
   @Test
+  public void testBigQueryNoTableQuerySourceInitSplit() throws Exception {
```

Comment:
Per discussion, we really need less code duplication in here. We're going to rewrite this code post-1.8 release.